src/screens/OtherProfile/index.js

```diff
@@ -114,9 +115,6 @@ const OtherProfile = () => {
     onBlockUserReportInfoSkipped,
     onShareUserButtonClicked
   } = eventTrack;
-  const [isCurrentFollowed, setIsCurrentFollowed] = React.useState(
-    getIsMeFollowingTargetStatus(params.data.other_id) || params?.data?.following
-  );
 
   const isSignedMessageEnabled = dataMain.isSignedMessageEnabled ?? true;
   const isAnonimityEnabled = dataMain.isAnonMessageEnabled && isSignedMessageEnabled;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the local state `isCurrentFollowed` and its initialization logic is a good change if this state is now being managed inside the `useOtherProfileScreenHooks()` hook. This helps to reduce redundancy and improve code maintainability.



<!-- This is an auto-generated comment by OSS CodeRabbit -->